qiskit/circuit/tools/pi_check.py

```diff
@@ -135,8 +135,15 @@ def normalize(single_inpt):
     complex_inpt = complex(inpt)
     real, imag = map(normalize, [complex_inpt.real, complex_inpt.imag])
 
+    jstr = '\\jmath' if output == 'latex' else 'j'
```

Comment:
This should also cover latex_source
```suggestion
    jstr = '\\jmath' if output in ['latex', 'latex_source'] else 'j'
```